pkgs/top-level/aliases.nix

```diff
@@ -305,6 +305,7 @@ mapAliases ({
   ec2_api_tools = ec2-api-tools; # Added 2021-10-08
   ec2-utils = amazon-ec2-utils; # Added 2022-02-01
 
+  elasticsearch-oss = throw "elasticsearch-oss has been removed because nixpkgs currently only has the unfree version of elasticsearch7"; # Added 2022-10-04
```

Comment:
Looks like there is no free variant anymore.

https://www.elastic.co/pricing/faq/licensing